src/d3d9/d3d9_swapchain.h

```diff
@@ -197,6 +198,24 @@ namespace dxvk {
 
     std::string GetApiName();
 
+    const Com<D3D9Surface, false>& GetFrontBuffer() const {
+      return m_backBuffers.back();
+    }
+
+    bool HasFrontBuffer() const {
+      if (m_presentParams.SwapEffect != D3DSWAPEFFECT_COPY)
```

Comment:
Does `m_backBuffers.count` need to be above 1 too? Or does this function purely have to rely on `SwapEffect`?

Reply:
This function is used to decide the size of `m_backBuffers`.